x-pack/plugins/searchprofiler/public/directives/profile_tree/index.html

```diff
@@ -2,12 +2,17 @@
   <div class="prfDevTool__shardDetails--dim">
     <span
       tooltip-placement="bottom"
-      tooltip="The cumulative time of all shards in the index. Note: this is not wall-clock time, as shards can execute in parallel.">
-      <h3>Cumulative Time: {{index.time[target] | msToPretty: 3}}</h3>
+      tooltip="{{:: 'xpack.searchProfiler.profileTree.cumulativeTimeTooltip' | i18n: { defaultMessage: 'The cumulative time of all shards in the index. Note: this is not wall-clock time, as shards can execute in parallel.' } }}">
+      <h3>{{:: 'xpack.searchProfiler.profileTree.cumulativeTimeTitle' | i18n: { defaultMessage: "Cumulative Time:" } }} {{ index.time[target] | msToPretty: 3 }}</h3>
     </span><br/>
   </div>
-  <div class="prfDevTool__shardDetails"  style="margin-left:5px">
-    <h3><b>Index:</b> {{index.name}}</h3>
+  <div class="prfDevTool__shardDetails" style="margin-left:5px">
+    <h3>
+      <b
+        i18n-id="xpack.searchProfiler.profileTree.indexTitle"
+        i18n-default-message="Index: "
+      ></b>{{index.name}}
```

Comment:
I believe this will work as well:
```
<b
        i18n-id="xpack.searchProfiler.profileTree.indexTitle"
        i18n-default-message="Index:"
></b>
{{index.name}}
```
And we don't have to rely on whether a vendor leave a space at the end of the string or not.